brd-android/kyc/src/main/java/com/fabriik/kyc/ui/features/countryselection/CountrySelectionViewModel.kt

```diff
@@ -58,17 +58,22 @@ class CountrySelectionViewModel(
             endState = { copy(initialLoadingVisible = false) },
             startState = { copy(initialLoadingVisible = true) },
             action = { kycApi.getCountries() },
-            callback = {
-                when (it.status) {
+            callback = { response ->
+                when (response.status) {
                     Status.SUCCESS -> {
-                        setState { copy(countries = it.data!!) }
+                        setState {
+                            val us = response.data!!.find { it.code == "US" }
+                            val arrayList = ArrayList(response.data!!)
+                            arrayList.add(0, us)
```

Comment:
You could use something like `sortedByDescending { it.isUsa }` instead. Also, I think this will duplicate USA in the list.

Reply:
I wanted to add US as a shortcut on the top but at the same time keep US on its own position just in case the user decides to scroll to find it in the alphabetical position.

Reply:
Ok, but if user searches for "united states of america" he gets two results which is quite confusing for user in my opinion? It would be beter if us would be in first place but not duplicated.

Reply:
I will approve this PR, but that is something that needs to be approved by design team and added to the ticket. Otherwise, QA team could reject your ticket because of that.

Reply:
fair enough

Reply:
> Ok, but if user searches for "united states of america" he gets two results which is quite confusing for user in my opinion? It would be beter if us would be in first place but not duplicated.

Good catch @lrupnikk, I will fix that situation where the search return two duplicated results using `distinct()` on the list.